docs/v0.3/sdk/go/fssrz-index.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: /fssrz/sdk-go
+---
+
+# Dagger SDK for Go
```

Comment:
We have previously agreed that this would be called "Go SDK for Dagger". However the convention seems to be "PRODUCT SDK for Go" rather than the other way around. Therefore I have changed it accordingly. Please let me know if you prefer the original version.

Reply:
Definitely one for @shykes.

Reply:
Good question...

* No matter what, when the context is unambiguously Dagger-specific, we should use the short form "Go SDK". For example in our navbar.

* I'm not sure what the correct long form is. But Stripe has a [Stripe iOS SDK](https://stripe.dev/stripe-ios/docs/) mentions, maybe we could have a "Dagger Go SDK" ?